src/shared/components/mutationMapper/MutationMapperDataStore.ts

```diff
@@ -196,6 +196,19 @@ export default class MutationMapperDataStore
         return countDuplicateMutations(this.tableDataGroupedByPatients);
     }
 
+    @computed
+    get dataSamples() {
+        return _.uniqBy(_.flatten(this.allData), m => m.sampleId).map(m => ({
+            sampleId: m.sampleId,
+            studyId: m.studyId,
+        }));
+    }
+
+    @computed
+    get dataPatients() {
+        return _.uniq(_.flatten(this.allData).map(m => m.patientId));
```

Comment:
since the word "patient" strongly suggests were' going to get the Patient type back, i would call this dataPatientIds.    The name "dataPatients" and "dataSamples" could probably be improved.  Like, we know it's data.  Lets make it more descriptive (even if there are other preexisting properties here which suffer from the same vagueness).

Reply:
renamed dataPatients to dataPatientIds and dataSamples to dataSampleIdentifiers